cluster-sim/cluster_sim/utils.py

```diff
@@ -88,26 +88,23 @@ class Persisted(object):
     filename = None
     default_state = {}
```

Comment:
unfortunately just resetting corosync and pacemaker state did not fix the issue. I've had to also ignore peers not in certain states during corosync configuration in order to work around this issue and to get the simulator tests passing reliably.